### DISCUSSION TITLE
想请问一下各位大佬是如何解决IOS设备证书问题的

### DISCUSSION THREAD

**Discussion by Entropy-Xu** [Lost-MSth/Arcaea-server#72] [Q&A] [answered]
我已经搭建好了服务端，在本地使用
客户端用HTTP Catcher DNS劫持，把arcapi-v2.lowiro.com解析到192.168.2.100的本地ip上
我在服务端上使用了自签证书，然后在服务端访问web界面的https是正常的
但是在客户端上访问时会出现不安全的提示，游戏内登录服务端也没有反应。

**Comment by Lost-MSth** [comment, marked as answer]
IP地址也能有可信证书吗？

其实问题不在这，客户端有证书校验啊()

**Reply by Entropy-Xu** [reply]
我是类似于改host那种把arcapi-v2.lowiro.com解析到本地ip上，不过客户端的话只能想办法干掉证书校验了吧

**Reply by Lost-MSth** [reply]
是的，客户端对特定域名检查证书是否正确

以及改host还是别用https了，首先ip地址应该是不会被信任的(除非直接证书装成安卓系统证书？)，其次客户端看到的地址依然是官方的，不可能对得上(除非去616那偷？)

**Reply by Entropy-Xu** [reply]
是所有客户端版本都有证书校验吗？大佬知道的最晚的没有证书校验的客户端版本是多少啊

**Reply by Lost-MSth** [reply]
3.6.0为分界线，所以推荐3.5.3(

**Reply by Entropy-Xu** [reply]
（真令人脸扁

**Comment by Entropy-Xu** [comment]
对了大佬，老版本的服务端(2.3.1)支持SSL吗

**Reply by Lost-MSth** [reply]
事实上不应该直接将Flask开成https模式，当然其实也不该直接用开发服务器

这种事交给专业的中间件来挂证书比较好，nginx apache什么的()


当然原则上是可以的运行的，就算我没有留配置也可以自己加个ssl

**Reply by Entropy-Xu** [reply]
了解

**Comment by Entropy-Xu** [comment]
对了大佬，4.1.0的服务端支持最新的4.1.3的客户端吗

**Reply by Lost-MSth** [reply]
啥，我穿越了？

客户端版本靠的近都可以无视差别，应该没什么改动的
